scenarios/throughput_stress.go

```diff
@@ -154,7 +167,7 @@ func (t *tpsExecutor) Run(ctx context.Context, info loadgen.ScenarioInfo) error
 	}
 
 	// Post-scenario, verify reported count from Visibility matches the expected count.
-	info.Logger.Info("Total workflows executed: ", t.state.WorkflowCount)
+	info.Logger.Info("Total workflows executed: ", t.state.CompletedIteration)
```

Comment:
Isn't this print wrong now? It's no longer total workflows?

Reply:
Ah true, I messed this up while going back and forth on this.

Reply:
```suggestion
	info.Logger.Info("Total workflows executed: ", t.state.WorkflowCount)
```